Automattic/wp-calypso
# SHARING - Add post publish sharing modal

In pe7F0s-gz-p2 we are adding a sharing modal after publishing a post. The purpose is to encourage bloggers to publish their post to social media and get views and engagement.

### Desktop

<img width="884" alt="CleanShot 2023-05-03 at 14 56 37@2x" src="https://user-images.githubusercontent.com/5634774/236016376-0cbe98b3-ba75-414e-b5aa-eb33b3f35ff4.png">

### Testing instructions

Sandbox a test site and 
Build and apply this version of the ETK to your sandbox with this command.
```
cd apps/editing-toolkit
yarn dev --sync
```
On your sandboxed blog, publish a new post
You should see the post publish sharing modal.

* clicking links should record tracks events
* The modal should not conflict with the other modals identified in https://github.com/Automattic/wp-calypso/pull/76615 or the jetpack [launchpad modal](https://github.com/Automattic/jetpack/blob/f60b8721717189e53129e856c75a9461a0d153fb/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/launchpad-save-modal.php)

#### Dismissing the modal.
The modal can be dismissed with the checkbox. The modal can be re-enabled from `wpsh` by running
```
update_blog_option( $site_id, 'sharing_modal_dismissed', false )
```
